ui/pages/settings/contact-list-tab/add-contact/add-contact.component.js

```diff
@@ -150,7 +156,7 @@ export default class AddContact extends PureComponent {
                     domainName={addressBookEntryName ?? domainName}
                     onClick={() => {
                       this.setState({
-                        selectedAddress: resolvedAddress,
+                        input: resolvedAddress,
```

Comment:
This leaves the input field editable after an ENS resolved address has been picked